libqtile/core/manager.py

```diff
@@ -1036,19 +1036,42 @@ def __str__(self):
                 return "".join([format % tuple(self.expandlist(row, n)) for row in self.rows])
 
         result = FormatTable()
-        result.add(["KeySym", "Mod", "Command", "Desc"])
+        result.add(["mode", "KeySym", "Mod", "Command", "Desc"])
         result.add([])
         rows = []
+
+        def recurse(ks: Union[int, str], kmm: Union[int, List[str]], k: Union[Key, KeyChord],
```

Comment:
Can we give this a better name? walk_key or something?

Reply:
Sure, no problem. But how on earth? I mean, this is a draft. Why are you reviewing that sooo early? :rofl:

Reply:
You made the PR, I'll review it if/when I have time. In my experience, early reviews are better than late reviews...